# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1522337345 14400
#      Thu Mar 29 11:29:05 2018 -0400
# Branch stable
# Node ID d298a0734d85f666e3b79f433aebc402eb88fd86
# Parent  1fe6002f68a76bdfc1ea5b55125489ca1626952a
avoid syntax error in configure script

* configure.ac: Quote uses of $octave_cv_qt_opengl_os_ok in test
commands.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2183,26 +2183,30 @@ AC_ARG_WITH([osmesa],
   [if test x"$withval" = xyes; then
      use_osmesa=yes
    else
      use_osmesa=no
    fi])
 
 dnl If no argument given, only use OSMesa when Qt is too old
 if test $use_osmesa = auto; then
-  if test $octave_cv_qt_opengl_os_ok = yes; then
+  ## Quote $octave_cv_qt_opengl_os_ok because it will only be set if
+  ## Qt is found.
+  if test "$octave_cv_qt_opengl_os_ok" = yes; then
     use_osmesa=no
   else
     use_osmesa=yes
   fi
 fi
 
 if test $use_osmesa = yes; then
   dnl Pre-declare warning message, and unset if library is found
-  if test $octave_cv_qt_opengl_os_ok = no; then
+  ## Quote $octave_cv_qt_opengl_os_ok because it will only be set if
+  ## Qt is found.
+  if test "$octave_cv_qt_opengl_os_ok" = no; then
     warn_osmesa="OSMesa library not found and Qt < 5.1.  Offscreen rendering with OpenGL will be disabled."
   else
     warn_osmesa="OSMesa library not found.  Offscreen rendering with FLTK will be disabled."
   fi
 
   OSMESA_LIBS="-lOSMesa"
   save_CPPFLAGS="$CPPFLAGS"
   save_LDFLAGS="$LDFLAGS"
